boot.php

```diff
@@ -116,13 +82,5 @@ function public_contact()
  */
```

Comment:
Please mark this method as deprecated
```suggestion
 * @deprecated since version 2022.12, use Core\Session:: getRemoteUser() instead
 */
```